andela/ah-frontend-poseidon
# #163081040: Design Dashboard.

**What does this PR do?**

Designs the dashboard the user will see once successfully logged in. 

**Description of the tasks to be completed?**
- Add navigation bar component. 
- Add navigation bar component to home view. 
- Add sidebar to dashboard. 

**How should this be manually tested?**
- Clone the repository
- git checkout ft-dashboard-163081040
- install dependencies using `yarn install`
- Run the app using `yarn start`
- The application should be running.
- Open browser and go to http://localhost:3000

**Screenshots**
![image](https://user-images.githubusercontent.com/17021137/51251791-2ad1b800-19ab-11e9-8d06-aff51725884d.png)


**What are the relevant pivotal tracker stories?**
[#163081040](https://www.pivotaltracker.com/story/show/163081040)
